hipcheck/src/analysis/mod.rs

```diff
@@ -5,49 +5,12 @@ pub mod score;
 use crate::{
 	config::{AttacksConfigQuery, CommitConfigQuery, PracticesConfigQuery},
 	data::git::GitProvider,
-	error::Result,
 	metric::MetricProvider,
-	plugin::QueryResult,
-	F64,
-};
-use std::{
-	default::Default,
 };
 
 /// Queries about analyses
 #[salsa::query_group(AnalysisProviderStorage)]
 pub trait AnalysisProvider:
 	AttacksConfigQuery + CommitConfigQuery + GitProvider + MetricProvider + PracticesConfigQuery
 {
```

Comment:
At this point `AnalysisProvider` is empty, time to get rid of the entire `trait`